docs/reading_events.md

```diff
@@ -0,0 +1,151 @@
+# Reading Events
+
+## Reading from a specific stream
+
+The simplest way to read a stream forwards is to supply a `PgEventstore::Stream` object.
+
+```ruby
+stream = PgEventstore::Stream.new(context: 'MYAwesomeContext', stream_name: 'User', stream_id: '1')
+PgEventstore.client.read(stream)
+# => [#<PgEventstore::Event 0x1>, #<PgEventstore::Event 0x1>, ...]
+```
+
+### max_count
+
+You can provide the `:max_count` option. This option determines how much records to return in a response. Default is `1000` and it can be changed via `:max_count` configuration setting(see [**"Configuration"**](configuration.md) chapter):
```

Comment:
```suggestion
You can provide the `:max_count` option. This option determines how many records to return in a response. Default is `1000` and it can be changed with the `:max_count` configuration setting (see [**"Configuration"**](configuration.md) chapter):
```